jax/_src/scipy/test_eigh.py

```diff
@@ -0,0 +1,49 @@
+"""Test for eigh.py."""
```

Comment:
Would you mind moving the tests into the jax/tests directory?

Reply:
Done